hardhat/README.md

```diff
@@ -64,17 +69,26 @@ This process downloads and starts an Optimistic Ethereum network of one node.
 1. Clone the [Optimism monorepo](https://github.com/ethereum-optimism/optimism).
 
    ```sh
-   git clone https://github.com/ethereum-optimism/optimism.git
+   git clone https://github.com/ethereum-optimism/optimism.git -b experimental
    ```
+   <!-- TEMO-OVM2.0 -->
 
-2. Start the Optimistic Ethereum node. This process downloads the images
+1. Copy a docker-compose file for OVM 2.0:
+
+   ```sh
+   cd optimism/ops
+   wget https://raw.githubusercontent.com/ethereum-optimism/optimism-tutorial/20211008-ovm20/ovm20-temp/docker-compose-ovm20.yml
```

Comment:
Is this really what we want as part of the devex of spinning up getting started project?